.github/workflows/testing.yml

```diff
@@ -72,6 +75,12 @@ jobs:
           # If we did not exit earlier, raise an error here
           exit 1
           ls
+      - name: Upload Results
+        if: failure()
+        uses: actions/upload-artifact@v3
+        with:
+          name: ${{github.job}}-${{github.run_number}}
+          path: ${{env.MESHPY_TESTING_DIR}}
```

Comment:
Can't you just set this to `path: ${{ github.workspace }}/tests/testing-tmp/`? Then all the other changes are not needed.

Reply:
I adapted it accordingly.